Source/utils/paths.cpp

```diff
@@ -75,7 +76,10 @@ const std::string &PrefPath()
 {
 	if (!prefPath)
 		prefPath = FromSDL(SDL_GetPrefPath("diasurgical", "devilution"));
-	return *prefPath;
+	if (FileExists("diablo.ini")) {
+		prefPath = "save/";

```

Comment:
Why is it adding `save` to the path, wouldn't existing save games normally be in the same folder?

Reply:
with the zero tier stuff it creates so much crap with the saves so it looks a lot nicer to me to save it in a save folder.

Reply:
But if we need to set it to be in the ./ so that it matches with Vanilla, I can switch it.

Reply:
To me adding save defeated the purpose of this PR. But we can and should move the ZeroTier files to a sub-folder.

Reply:
```suggestion
		prefPath = "./";

```